### ISSUE TITLE
⛔ ♿ 🐞 Bug Report: Can't add aria-label when using site logo and you add a link

### ISSUE THREAD

**Issue by hezyse** [elementor/elementor#22491]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

HI,
when I'm adding site logo and add a link the <a> tag contain only the url.
there no way to add aria-label 
I tried to add it in advanced tab but it won't affect it.

### Steps to reproduce

just add a site logo and add a link

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.3.36
	PHP Version: 7.4.33
	PHP Memory Limit: 768M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.1
	Site URL: https://www.barakdis.co.il
	Home URL: https://www.barakdis.co.il
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 40M
	Max Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: he-IL
	Timezone: Asia/Jerusalem
	Admin Email: webdhm@gmail.com
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: he_IL
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.1.6
		Author: WP Engine

	call-and-whatsapp-button
		Version: 1.0.3
		Author: Hezy Sela

	Category Posts in Custom Menu
		Version: 1.3.0
		Author: Diana Koenraadt

	Classic Editor
		Version: 1.6.3
		Author: WordPress Contributors

	Contact Form 7
		Version: 5.7.6
		Author: Takayuki Miyoshi

	Custom Admin Login Page
		Version: 1.0.1
		Author: Hezy Sela

	Disable REST API
		Version: 1.7
		Author: Dave McHale

	Disable XML-RPC
		Version: 1.0.1
		Author: Philip Erb

	Elementor
		Version: 3.13.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.1
		Author: Elementor.com

	Embed Plus YouTube WordPress Plugin
		Version: 14.1.6.2
		Author: Embed Plus YouTube Plugin Team

	Imagify
		Version: 2.1.1
		Author: Imagify – Optimize Images & Convert WebP

	WP Rocket
		Version: 3.13.2
		Author: WP Media

	Yoast SEO
		Version: 20.7
		Author: Team Yoast


== Must-Use Plugins ==
	SSO
		Version: 0.1
		Author: Garth Mortensen, Mike Hansen


== שימוש באלמנטים ==
	
	archive : 1
		archive-posts : 1
		theme-archive-title : 1
	footer : 1
		image : 1
		text-editor : 1
	header : 1
		breadcrumbs : 1
		heading : 1
		image : 1
		nav-menu : 1
		search-form : 1
		theme-site-logo : 1
	wp-post : 0
		gallery : 34
	wp-page : 3
		gallery : 1
		heading : 1
		portfolio : 1
	single-post : 2
		theme-post-content : 2
		theme-post-featured-image : 1
		theme-post-title : 1
	error-404 : 1
		button : 1
		heading : 3
		icon : 1
		image : 1
		spacer : 2


== הגדרות ==
	
	cpt_support: post, page
	allow_tracking: yes
	font_display: swap


== תכונות ==
	Custom Fonts: 0
	Custom Icons: 0

== אינטגרציות ==
	


== הניסויים של אלמנטור ==
	פלט DOM שעבר אופטימיזציה: פעיל כברירת מחדל
	טעינת נכסים משופרת: פעיל כברירת מחדל
	טעינת CSS משופרת: פעיל כברירת מחדל
	איקון פונטס בשורה: לא פעיל כברירת מחדל
	שיפורי נגישות: פעיל כברירת מחדל
	Additional Custom Breakpoints: פעיל כברירת מחדל
	admin_menu_rearrangement: לא פעיל כברירת מחדל
	פלקסבוקס קונטיינר: פעיל
	שדרג ספריית Swiper: פעיל כברירת מחדל
	גריד קונטיינר: פעיל
	ברירת מחדל לממשק החדש של בונה התבנית: פעיל כברירת מחדל
	עיצוב כותרת עליונה ותחתונה: פעיל כברירת מחדל
	תפריט עליון של העורך: פעיל
	דפי נחיתה: פעיל כברירת מחדל
	אלמנטים מקוננים: פעיל
	טעינה עצלה של תמונות רקע: לא פעיל כברירת מחדל
	מעבר עמוד: פעיל כברירת מחדל
	פתקים: פעיל כברירת מחדל
	לולאה: פעיל כברירת מחדל
	לידים: פעיל כברירת מחדל
	קפיצת גלילה: פעיל כברירת מחדל
	תפריט: פעיל


== יומן שינויים ==
	
PHP: showing 5 of 5PHP: 2023-05-07 19:49:56 [notice X 18][/home/barakdisco/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-17 17:16:49 [error X 1][G:\websites\barakdis\barkdis_website\wp-content\plugins\elementor\includes\template-library\manager.php::512] Out of memory (allocated 62914560) (tried to allocate 32768 bytes) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-17 19:03:31 [error X 7][G:\websites\barakdis\barkdis_website\wp-content\plugins\elementor-pro\modules\notes\user\delete-user.php::69] Out of memory (allocated 6291456) (tried to allocate 5242880 bytes) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-17 20:14:06 [error X 1][G:\websites\barakdis\barkdis_website\wp-content\plugins\elementor-pro\modules\notes\user\delete-user.php::69] Out of memory (allocated 14680064) (tried to allocate 5242880 bytes) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-18 10:19:10 [notice X 2][G:\websites\barakdis\barkdis_website\wp-content\plugins\elementor-pro\core\app\modules\site-editor\data\endpoints\templates.php::150] Undefined index: condition_type [array (
  'trace' => '
#0: G:\websites\barakdis\barkdis_website\wp-content\plugins\elementor-pro\core\app\modules\site-editor\data\endpoints\templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: G:\websites\barakdis\barkdis_website\wp-content\plugins\elementor-pro\core\app\modules\site-editor\data\endpoints\templates.php(120): class type array_map()
#3: G:\websites\barakdis\barkdis_website\wp-content\plugins\elementor-pro\core\app\modules\site-editor\data\endpoints\templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: G:\websites\barakdis\barkdis_website\wp-content\plugins\elementor\data\base\endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]

Log: showing 10 of 102023-05-12 15:34:13 [info] elementor::elementor_updater Started 
2023-05-12 15:34:13 [info] אלמנטור/Upgrades - _on_each_version Start  
2023-05-12 15:34:13 [info] אלמנטור/Upgrades - _on_each_version Finished 
2023-05-12 15:34:13 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '3.12.2',
  'to' => '3.13.2',
)]
2023-05-12 15:34:13 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'אלמנטור',
  'from' => '3.12.2',
  'to' => '3.13.2',
)]
2023-05-12 15:35:33 [info] elementor-pro::elementor_pro_updater Started 
2023-05-12 15:35:33 [info] אלמנטור פרו/Upgrades - _on_each_version Start  
2023-05-12 15:35:33 [info] אלמנטור פרו/Upgrades - _on_each_version Finished 
2023-05-12 15:35:33 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '3.12.3',
  'to' => '3.13.1',
)]
2023-05-12 15:35:33 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '3.12.3',
  'to' => '3.13.1',
)]

JS: showing 2 of 2JS: 2023-05-16 20:38:04 [error X 11][https://www.barakdis.co.il/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.13.2:3:795450] Cannot read properties of undefined (reading 'localized') 
JS: 2023-05-17 19:58:26 [error X 1][https://www.barakdis.co.il/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.13.2:3:907878] elementorFrontend is not defined 



== אלמנטור - Compatibility Tag ==
	
	Elementor Pro: תאימות לא הוגדרה

== אלמנטור פרו - Compatibility Tag ==
	

				
````

</details>

**Comment by perks98** [comment]
@hezyse have you tried changing the site logo link from site url option to custom url but then use the dynamic tag to choose the site url again but you should now have the option for link options to add custom attributes? Video below if that helps:

https://github.com/elementor/elementor/assets/55829113/d5d0d259-db08-45cb-b996-2f8424e7bf41

**Comment by hezyse** [comment]
@perks98 hi,
that helped
Thanks a lot
